jenkins-pipelines/longevity-cdc-8h-multi-dc-topology-changes.jenkinsfile

```diff
@@ -6,6 +6,7 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 longevityPipeline(
     backend: 'aws',
     region: '''["eu-west-1", "us-east-1"]''',
+    availability_zone: 'a,b',
```

Comment:
Let's move to the test yaml and set the AZ and RF to be the same.

Reply:
this test has 4 nodes per DC, so it will make cluster unbalanced from the start in that case. 
I change multidc case to `longevity-counters-multidc.yaml`